addons/quest_system/quest_manager.gd

```diff
@@ -173,6 +173,33 @@ func set_quest_property(quest_id: int, property: String, value: Variant) -> void
 
 	# Finally we set the value
 	quest.set(property, value)
+	
+func get_quest_property(quest_id: int, property: String) -> Variant:
+	var quest: Quest = null
+
+	# Find the quest
+	for pools in get_all_pools():
+		if pools.get_quest_from_id(quest_id) != null:
+			quest = pools.get_quest_from_id(quest_id)
+
+	if quest == null: return null
+
+	# Now check if the quest has the property
+
+	# First if the property is null -> we return
+	if property == null: return null
+
+	var was_property_found: bool = false
+	# Then we check if the property is present
+	for p in quest.get_property_list():
+		if p.name == property:
+			was_property_found = true
+			break
+
+	# Return if the property was not found
+	if not was_property_found: return null
+	# Finally we get the value
+	return quest.get(property)
```

Comment:
Nice catch! This method is indeed something that is missing from the main quest system api. I'd suggest you making a PR on the addon repo as this is something that could be added there too (and to make it easier to update when a new version of Quest System comes out).

Reply:
Sure thing! I'll do!